### DISCUSSION TITLE
Useful links (possibly put into the design system)

### DISCUSSION THREAD

**Discussion by michaelcattell** [dwp/design-system#196] [Ideas]
A list of useful links that may appear in design system useful for all roles:

**Comment by michaelcattell** [comment]
Rec'd from Jon Houghton:
(these links appear on the design manual)

**Flow templates on GOV.UK Design System**
https://design-system.service.gov.uk/community/resources-and-tools/
**GOV.Flow**
https://github.com/charlesrt/gov-flow
**GOV.Flow in MIRO**
https://github.com/paulmsmith/govuk-designsystem-flow-diagram-miro
**Vicky's GOV.UK elements in MURAL**
https://github.com/vickytnz/govuk-elements-mural
**Sketch wireframing kit**
https://github.com/abbott567/sketch_wireframing_kit
**Monster list of books**
https://airtable.com/universe/expqM3OWZoJkjl7wy/the-monster-list-of-ux-books
**Design Systems**
https://github.com/ctdesign/gov-design-systems-list

**Comment by HelenOsg** [comment]
DWP style guide - https://intranet.dwp.gov.uk/section/communications/communication-standards/written
GDS style guide - https://www.gov.uk/guidance/style-guide
